en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,68 @@
 
-__Location data.__ Mozilla Subscription Services receives your IP address when you sign up for and use the service(s) and it is collected as part of our servers logs for 90 days; we use the IP address in order to approximate your location because Mozilla Subscription Services availability, pricing and offers may depend on your country. We also use it for fraud and abuse prevention purposes across our products.
+__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you’re using the Mozilla VPN service in any circumstance. Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
 
-__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you're using the Mozilla VPN service.  Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
+### Mozilla Monitor {: #monitor }
+
+__Interaction data.__ Mozilla receives data about your interactions with the Services. For instance, when you log in and out and the preferences you set; learn more about [Monitor interaction data](https://dictionary.telemetry.mozilla.org/apps/monitor_frontend). More generally, Mozilla may also use cookies, device information, and IP addresses, along with clear GIFs, cookies and third party services to help us understand in the aggregate how users engage with our products, services, communications, websites, online campaigns, snippets, devices, and other platforms. We use:
+
+* [HaveIbeenpwned.](https://haveibeenpwned.com/) HaveIbeenpwned receives a hashed version of your email address in order to perform a scan of where your personal data may have been compromised. HaveIbeenpwned keeps this personal data to continuously scan for new breaches; you can request removal of your email address from HaveIbeenpwned via their [opt-out mechanism](https://haveibeenpwned.com/OptOut).
+  
+* [OneRep.](https://onerep.com/) If you are located in the United States and have a Monitor Premium Subscription, OneRep receives your first and last name, email address, phone number, physical address and date of birth in order to scan data broker sites to find your personal data and request its removal. OneRep keeps your personal data until you end your Monitor subscription in order to check whether your information shows up on additional sites, or has reappeared on the sites you’ve already been removed from.
```

Comment:
Open question: Isn't it going to be Monitor Plus by the time this document is valid? Also, I think subscription should be lowercase here.